src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -287,6 +292,31 @@ export const gravityStitchingEnvironment = (
         },
       },
       MarketingCollection: {
+        thumbnailImage: {
+          fragment: gql`
+          ... on MarketingCollection {
+            image_url: thumbnail
+          }
+          `,
+          resolve: async ({ image_url }, args, context, info) => {
+            if (image_url) {
+              context.imageData = {
+                image_url,
+              }
```

Comment:
**question**: maybe it would be a good idea to use `normalizeImageData` by analogy as [here](https://github.com/artsy/metaphysics/blob/a8f7199c29da02ad89233ca01d4aabaa253b98fc/src/schema/v2/article/index.ts#L385)?